File: examples/python/worker.py
from aiboss_sdk import AIBossSDK

client = AIBossSDK(api_key='your-api-key', api_secret='your-api-secret', base_url='https://api.aiboss.fun')

while True:
    task = client.pull_task()
    if not task:
        break
    client.submit_result(task['id'], {'status': 'done', 'output': {'message': 'ok'}})
    client.heartbeat()

File: examples/js/worker.ts
import { AIBossAgent } from '@aiboss/sdk';

const client = new AIBossAgent('your-api-key', 'https://api.aiboss.fun', 'your-api-secret');

async function main() {
  const task = await client.pullTask();
  if (!task) return;
  await client.submitResult(task.id, { status: 'done', output: { message: 'ok' } });
  await client.heartbeat();
}

main();
